src/java.base/share/classes/java/lang/Class.java

```diff
@@ -3827,10 +3831,12 @@ public Class<?> getNestHost() {
      * @param c the class to check
      * @return {@code true} if this class and {@code c} are members of
      * the same nest; and {@code false} otherwise.
+     * @throws NullPointerException if {@code c} is {@code null}
      *
      * @since 11
      */
     public boolean isNestmateOf(Class<?> c) {
+        Objects.requireNonNull(c);
```

Comment:
So the behavior change here is calling isNestmateOf(null) on a class object for a primitive or array class will throw NPE when it didn't throw it previously.  Technically an incompatible change but I agree it's in the weeds when it comes to impact.

Reply:
Removed erroneous comment

Reply:
Yes you are right.